src/commands/database-rules-canary.ts

```diff
@@ -0,0 +1,23 @@
+import { Command } from "../command";
+import * as logger from "../logger";
+import * as requireInstance from "../requireInstance";
+import { requirePermissions } from "../requirePermissions";
+import * as metadata from "../database/metadata";
+
+export default new Command("database:rules:canary <ruleset_id>")
+  .description("mark a staged ruleset as the canary ruleset")
+  .option(
+    "--instance <instance>",
+    "use the database <instance>.firebaseio.com (if omitted, uses default database instance)"
+  )
+  .before(requirePermissions, ["firebasedatabase.instances.update"])
+  .before(requireInstance)
+  .action(async (rulesetId: string, options: any) => {
+    const oldLabels = await metadata.getRulesetLabels(options.instance);
+    const newLabels = {
+      stable: oldLabels.stable,
+      canary: rulesetId,
+    };
+    await metadata.setRulesetLabels(options.instance, newLabels);
+    return newLabels;
```

Comment:
I think my last request about these returned values - do they have well known types that could be passed back to the client? That would make it real nice when someone uses the CLI as a library.

Reply:
For a certain definition of "well known". This has the same return type as `metadata.getRulesetLabels` --- a map from ruleset label ("stable" | "canary") to ruleset id (a uuid string).

Reply:
Okay. We can always improve on this later. It may be nice to return a typed thing, but I actually don't think the types carry through to scripts anyway (because of how `action` works), so this LGTM.